examples/07-EMIT/interference_gui.py

```diff
@@ -235,19 +298,37 @@ def open_file_dialog(self):
             self.radio_specific_levels.setEnabled(True)
             self.protection_results_btn.setEnabled(True)
             self.interference_results_btn.setEnabled(True)
-    
+
+    ###############################################################################
+    # Change design selection
+    # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    # Refresh the warning messages when the selected design changes
```

Comment:
Just realized that a sentence in a new section of an example should have concluding punctuation. Would you please check these to add a period if needed? Thanks.